packages/terra-navigation-prompt/docs/implementation-guide/IMPLEMENTATION_GUIDE_PART_0.md

```diff
@@ -0,0 +1,81 @@
+# Terra Navigation Prompt - Implementation Guide
+
+Assume we have built a simple component, the FormSwitcher, that toggles between showing two different stateful components, Form 1 and Form 2. Users of the FormSwitcher have recently complained that they can switch between forms before submitting their in progress form, losing their data in the process. We want to implement the NavigationPrompt and NavigationPromptCheckpoint to give our users more control over the FormSwitcher's navigation.
+
+> Note: This implementation guide features examples and use cases for the various APIs provided by the NavigationPrompt and NavigationPromptCheckpoint.
+> Please review your individual requirements to see which APIs are necessary for your specific needs.
```

Comment:
> ‼️ Please review your individual requirements to see which APIs are necessary for your specific needs.